specs/core/0_beacon-chain.md

```diff
@@ -805,17 +816,24 @@ def get_randao_mix(state: BeaconState,
     return state.latest_randao_mixes[epoch % LATEST_RANDAO_MIXES_LENGTH]
 ```
 
-### `get_active_index_root`
+### `get_active_registry_root`
```

Comment:
This name implies to me that it is going to get an active registry root from `latest_active_registry_roots` similar to the function `get_randao_mix`.

I actually think the term `compute_` is more appropriate here. We compute something to put it into the state.